internal/tiltfile/skylarkTypes.go

```diff
@@ -229,3 +230,32 @@ func (gitRepo) Hash() (uint32, error) {
 func badTypeErr(b *skylark.Builtin, ex interface{}, v skylark.Value) error {
 	return fmt.Errorf("%v expects a %T; got %T (%v)", b.Name(), ex, v, v)
 }
+
+func (gr gitRepo) Attr(name string) (skylark.Value, error) {
+	switch name {
+	case "path":
+		return skylark.NewBuiltin(name, fullRepoPath).BindReceiver(gr), nil
+	default:
+		return nil, nil
+	}
+
+}
+
+func (gr gitRepo) AttrNames() []string {
+	return []string{"path"}
+}
+
+func fullRepoPath(thread *skylark.Thread, fn *skylark.Builtin, args skylark.Tuple, kwargs []skylark.Tuple) (skylark.Value, error) {
```

Comment:
hmm...if you make this a method of `gitRepo`, you will get the binding for free, rather than having to bind and unbox?

Reply:
oh! Let me try that.